tests/src/core/importType.js

```diff
@@ -96,7 +98,6 @@ describe('importType(name)', function () {
   })
 
   it("should return 'unknown' for any unhandled cases", function() {
-    expect(importType('@malformed', context)).to.equal('unknown')
```

Comment:
why does this need to be removed?

Reply:
`@malformed` is not anymore an unhandled case. This is the purpose of this PR to accept `@foo` pattern.
The test has been replaced by [this one](https://github.com/benmosher/eslint-plugin-import/pull/1493/files#diff-410595ac51ca4f4f193da8d5704b1b0bR59).